README.md

```diff
@@ -71,6 +71,30 @@ cd spark-connectors
 ls -lhR ~/.m2/repository/io/pravega/pravega-connectors-spark
 ```
 
+### Test the Spark Connector
+As Pravega 0.9 runs on Java 11+ while spark 2.4 runs on Java 8, it needs to connect to a standalone Pravega instance for the test.
+1. Start Pravega 0.9 using Java 11.
+    1. Checkout the latest source code from [Pravage branch r0.9](https://github.com/pravega/pravega/tree/r0.9)
+       ```
+       git clone https://github.com/pravega/pravega.git
+       cd pravega
+       git checkout r0.9 
+       ```
+    2. Start Pravega standalone instance
+       ```
+       ./gradlew startStandalone
+       ```
+
+2. Run spark-connectors/gradlew test using Java 8.
+   By default, the test will connect to Pravega instance listening on tcp://localhost:9090.
+    ```
+    ./gradlew clean build
```

Comment:
Change this line to show how to use an explicit version of Java.
```
JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 ./gradlew ...
```

Reply:
done